manifests/init.pp

```diff
@@ -325,9 +329,11 @@
       register_in_foreman   => $register_in_foreman,
       foreman_base_url      => $foreman_url,
       registered_proxy_url  => "https://${capsule_fqdn}:${capsule::foreman_proxy_port}",
+      trusted_hosts         => undef,
```

Comment:
Really, we should've had a PR to address this when https://github.com/theforeman/puppet-foreman_proxy/commit/952f6bd7c83470cdd25f38f6d768bc4bf10e6e95 was merged, but it doesn't matter now.

All the cats I'm herding now are going to end up in katello-installer in a single PR.